*** ./controller.yaml.orig	2016-11-14 23:30:38.000000000 -0500
--- ./controller.yaml	2018-02-12 20:43:34.336697787 -0500
***************
*** 101,106 ****
--- 101,107 ----
              -
                type: interface
                name: nic1
+               ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                use_dhcp: false
                addresses:
                  -
***************
*** 119,146 ****
                dns_servers: {get_param: DnsServers}
                members:
                  -
!                   type: ovs_bond
                    name: bond1
!                   ovs_options: {get_param: BondInterfaceOvsOptions}
                    members:
                      -
                        type: interface
                        name: nic2
                        primary: true
                      -
                        type: interface
                        name: nic3
                  -
                    type: vlan
                    device: bond1
                    vlan_id: {get_param: ExternalNetworkVlanID}
-                   addresses:
-                     -
-                       ip_netmask: {get_param: ExternalIpSubnet}
-                   routes:
-                     -
-                       default: true
-                       next_hop: {get_param: ExternalInterfaceDefaultRoute}
                  -
                    type: vlan
                    device: bond1
--- 120,142 ----
                dns_servers: {get_param: DnsServers}
                members:
                  -
!                   type: linux_bond
                    name: bond1
!                   bonding_options: {get_param: BondInterfaceOvsOptions}
                    members:
                      -
                        type: interface
                        name: nic2
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                        primary: true
                      -
                        type: interface
                        name: nic3
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                  -
                    type: vlan
                    device: bond1
                    vlan_id: {get_param: ExternalNetworkVlanID}
                  -
                    type: vlan
                    device: bond1
***************
*** 169,174 ****
--- 165,186 ----
                    addresses:
                      -
                        ip_netmask: {get_param: TenantIpSubnet}
+             -
+               type: ovs_bridge
+               name: br-extnet
+               addresses:
+                 -
+                   ip_netmask: {get_param: ExternalIpSubnet}
+               routes:
+                 -
+                   default: true
+                   next_hop: {get_param: ExternalInterfaceDefaultRoute}
+               members:
+                 -
+                   type: interface
+                   name: nic4
+                   ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
+                   use_dhcp: false
                  # Uncomment when including environments/network-management.yaml
                  # If setting default route on the Management interface, comment
                  # out the default route on the External interface. This will
*** ./ceph-storage.yaml.orig	2017-02-15 14:24:34.860455403 -0500
--- ./ceph-storage.yaml	2018-02-12 20:42:44.152135414 -0500
***************
*** 101,106 ****
--- 101,107 ----
              -
                type: interface
                name: nic1
+               ethtool_opts: "-G ${DEVICE} rx 4096 tx 40966"
                use_dhcp: false
                dns_servers: {get_param: DnsServers}
                addresses:
***************
*** 129,137 ****
--- 130,140 ----
                      -
                        type: interface
                        name: nic2
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                        primary: true
                      -
                        type: interface
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                        name: nic3
                  -
                    type: vlan
*** ./cinder-storage.yaml.orig	2017-02-15 14:24:34.861455394 -0500
--- ./cinder-storage.yaml	2018-02-12 20:45:23.160750729 -0500
***************
*** 101,106 ****
--- 101,107 ----
              -
                type: interface
                name: nic1
+               ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                use_dhcp: false
                dns_servers: {get_param: DnsServers}
                addresses:
***************
*** 129,138 ****
--- 130,141 ----
                      -
                        type: interface
                        name: nic2
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                        primary: true
                      -
                        type: interface
                        name: nic3
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                  -
                    type: vlan
                    device: bond1
*** ./compute.yaml.orig	2017-11-03 17:29:12.000000000 -0400
--- ./compute.yaml	2018-02-12 20:43:06.528940210 -0500
***************
*** 101,106 ****
--- 101,107 ----
              -
                type: interface
                name: nic1
+               ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                use_dhcp: false
                dns_servers: {get_param: DnsServers}
                addresses:
***************
*** 129,138 ****
--- 130,141 ----
                      -
                        type: interface
                        name: nic2
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                        primary: true
                      -
                        type: interface
                        name: nic3
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                  -
                    type: vlan
                    device: bond1
*** ./swift-storage.yaml.orig	2017-02-15 14:24:34.865455360 -0500
--- ./swift-storage.yaml	2018-02-12 20:44:18.023317283 -0500
***************
*** 101,106 ****
--- 101,107 ----
              -
                type: interface
                name: nic1
+               ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                use_dhcp: false
                dns_servers: {get_param: DnsServers}
                addresses:
***************
*** 129,138 ****
--- 130,141 ----
                      -
                        type: interface
                        name: nic2
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                        primary: true
                      -
                        type: interface
                        name: nic3
+                       ethtool_opts: "-G ${DEVICE} rx 4096 tx 4096"
                  -
                    type: vlan
                    device: bond1
