--- driver.py.orig	2016-11-21 15:19:53.000000000 -0500
+++ driver.py	2017-01-24 18:58:35.000000000 -0500
@@ -3818,6 +3818,9 @@
             # instance as an optimisation
             return GuestNumaConfig(allowed_cpus, None, None, None)
         else:
+            if CONF.libvirt.virt_type == "qemu":
+                LOG.info('Disabling NUMA in qemu/TCG mode')
+                return GuestNumaConfig(None, None, None, None)
             if topology:
                 # Now get the CpuTune configuration from the numa_topology
                 guest_cpu_tune = vconfig.LibvirtConfigGuestCPUTune()
